source/_docs/metrics.md

```diff
@@ -2,13 +2,13 @@
 title: Metrics in the Site Dashboard
 description: Measure your site's traffic with the Metrics tool, found in the Live environment of the Site Dashboard.
 tags: [performance, logs]
-draft: true
 ---
 ## Access Metrics
 1. Navigate to **<span class="glyphicons glyphicons-charts"></span> Metrics** within the **<span class="glyphicons glyphicons-cardio"></span> Live** environment of your Site Dashboard.
 2. Toggle available date ranges by clicking **Day**, **Week**, or **Month**:
   ![Charts for pages served and visits within the Metrics tool of the Site Dashboard](/source/docs/assets/images/dashboard/metrics-graphs.png)
 
+
```

Comment:
Preserving @alexfornuto's original comment in #3448: 
> Consider expanding this screenshot to show the nav bar on the left with **Metrics** selected. This will make the screenshot useful for both steps.